.github/workflows/ci.yml

```diff
@@ -13,14 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby:
-        - ruby-2.1
-        - ruby-2.3
-        - ruby-2.4
-        - ruby-2.5
-        - ruby-2.6
-        - ruby-2.7
-        - jruby
+        ruby: [2.1, 2.3, 2.4, 2.5, 2.6, 2.7, jruby]
```

Comment:
@dkubb I think the commmit message is wrong: Both before this commit and after commit we used arrays. We just now use a different syntax to express the same array.

I'd suggest a commit message of:

> Refactor github actions to inline array syntax.